app/build.sh

```diff
@@ -78,6 +78,11 @@ With "--minishift" Minishift can be initialized and installed with Syndesis
     --watch                   Watch startup of pods
 -i  --image-mode <mode>       Which templates to install: "docker" for plain images, "openshift" for image streams
                               (default: "openshift")
+
+With "--dev" common development tasks are simplified
+
+    --debug <name>           Setup a port forwarding to <name> pod (example: rest)
+
```

Comment:
one space to less indent ;-) (yeah, they call me the "German" for some reason :)

Reply:
Now I have to create another PR for the missing space 😱

Reply:
Now I have to create another PR for the missing space 😱

Reply:
no, its fine. you can even fix this on upstream/master via the webui.

2s2f